playwright-tests/playwright.browserstack.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig } from '@playwright/test';
+import { getCdpEndpoint } from './browserstack.config';
+import { baseObject } from './playwright.base.config';
+
+export default defineConfig({
+	...baseObject,
+
+	globalSetup: require.resolve('./global-setup.ts'),
+	globalTeardown: require.resolve('./global-teardown.ts'),
```

Comment:
What's the reason for not just putting these methods in this file? They seem pretty coupled together?

Reply:
require.resolve here is just returning a string path to the files themselves and not the file contents. This appears to be the expected behaviour of the `defineConfig` function from 'playwright/test'